### DISCUSSION TITLE
Command/Ctrl-K "shortcut bar" modal window doesn't respect the Esc key

### DISCUSSION THREAD

**Discussion by hibachrach** [community/community#101506] [Accessibility] [answered]
### Select Topic Area

Bug

### Body

After hitting <kbd>Cmd-K</kbd>, I get shown a modal window, but hitting <kbd>Esc</kbd> doesn't close it. The only way to close it is to hit <kbd>Cmd-K</kbd> again. As far as I can tell, this goes against [ARIA guidelines for accessibility regarding modals](https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/).

**Comment by georgetoloraia** [comment]
@hibachrach 

- Browser/OS: 
- [Include details about the browser and operating system where the issue was observed]

**Reply by hibachrach** [reply]
* Browser: Firefox 122.0b9
* macOS Sonoma (14.0)

**Reply by ghostinhershell** [reply]
Thanks for hopping in and lending a helping hand here @georgetoloraia 🚀

**Comment by hibachrach** [comment, marked as answer]
This turned out to be caused by the Vimium extension on my browser--disabling it removes this issue.

**Reply by georgetoloraia** [reply]
OK

**Reply by ghostinhershell** [reply]
Really glad you were able to pin down what was causing the issue here @hibachrach 🎉 . I'll mark your response as the answer just so that anyone else who may run into this issue can find this. Thanks!

**Reply by georgetoloraia** [reply]
Thank you too
